packages/default-nav/management/default_navigation.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { i18n } from '@kbn/i18n';
+import type { NodeDefinitionWithChildren } from '@kbn/core-chrome-browser';
+import type { DeepLinkId } from '@kbn/deeplinks-management';
+
+export type NavigationID =
+  | 'rootNav:management'
+  | 'root'
+  | 'integration_management'
+  | 'stack_management'
+  | 'ingest'
+  | 'data'
+  | 'alerts_and_insights'
+  | 'kibana';
+
+export type ManagementNodeDefinition = NodeDefinitionWithChildren<DeepLinkId, NavigationID>;
+
+export const defaultNavigation: ManagementNodeDefinition = {
+  id: 'rootNav:management',
+  title: i18n.translate('defaultNavigation.management.sectionLabel', {
+    defaultMessage: 'Management',
+  }),
+  icon: 'gear',
+  children: [
+    {
+      id: 'root',
+      title: '',
+      children: [
+        {
+          link: 'monitoring',
+        },
+      ],
+    },
+    {
+      id: 'integration_management',
+      title: i18n.translate('defaultNavigation.management.integrationManagement', {
+        defaultMessage: 'Integration management',
+      }),
+      children: [
+        {
+          link: 'integrations',
+        },
+        {
+          link: 'fleet',
+        },
+        {
+          link: 'osquery',
+        },
+      ],
+    },
+    {
+      id: 'stack_management',
```

Comment:
@alisonelizabeth @sabarasaba This is the default "preset" navigation for stack management exposed in the package. I do see that all stack management deeplinks are marked as `hidden` and as there are no link the whole "Stack management" section is stripped from the management menu. I wanted to confirm with you that it is the expected behaviour in serverless. Cheers! 👍 

<img width="617" alt="Screenshot 2023-06-13 at 11 38 30" src="https://github.com/elastic/kibana/assets/2854616/a3a09cb0-7adf-4038-8e01-d8637118128a">



There is no "Stack management" under "Management"

<img width="241" alt="Screenshot 2023-06-13 at 11 37 04" src="https://github.com/elastic/kibana/assets/2854616/bef86fb5-b29e-46c6-87b8-b75dcb7da47b">

Reply:
Filed https://github.com/elastic/kibana/issues/160349